pom.xml

```diff
@@ -92,33 +155,52 @@
     <pluginManagement>
       <plugins>
         <plugin>
-          <groupId>com.mycila</groupId>
-          <artifactId>license-maven-plugin</artifactId>
-          <version>${maven.licence.plugin}</version>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>${maven.compiler.plugin}</version>
           <configuration>
-            <header>LICENSE.header</header>
-            <encoding>${project.build.sourceEncoding}</encoding>
-            <skipExistingHeaders>true</skipExistingHeaders>
-            <mapping>
-              <java>SLASHSTAR_STYLE</java>
-            </mapping>
-            <includes>
-              <include>sec/test/java</include>
-            </includes>
-            <properties>
-              <year>${project.inceptionYear}</year>
-              <name>${project.organization.name}</name>
-            </properties>
+            <source>${maven.compiler.source}</source>
+            <target>${maven.compiler.target}</target>
           </configuration>
-          <executions>
-            <execution>
-              <goals>
-                <goal>check</goal>
-              </goals>
-            </execution>
-          </executions>
         </plugin>
       </plugins>
     </pluginManagement>
   </build>
+
+  <dependencyManagement>
+    <dependencies>
+      <!-- Internal dependencies -->
+      <dependency>
+        <groupId>org.gwtproject.geolocation</groupId>
+        <artifactId>gwt-geolocation</artifactId>
+        <version>${revision}</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <repositories>
```

Comment:
this needs to be removed from here, and only exist in the test project(s) as necessary